scripts/seedsDB.js

```diff
@@ -58,49 +58,49 @@ db.Project.remove({})
   // This file empties the risks collection and inserts the books below
   const riskSeed = [   
     {
-        title: "title1",
+        title: "London Eye",
         riskId: "riskId1",
-        description:"discription1",
+        description: "Annual maintenance",
         designDiscipline: "designDiscipline1",
         status: 2,
         location: {
-            lat: 52.505641,
-            lng: -1.923124,
+            lat: 50.353626,
+            lng: 1.6235,
           },
         likelihood:4,
         severity:4,
         risk:8,
-        projectId:"5f4eec89cc853f32ec03243e"         
+        projectId: "5f59f053a94acd5aa0e4ebeb"         
     },
     {
-        title: "title2",
+        title: "London Bridge",
         riskId: "riskId2",
-        description:"discription2",
+        description: "Road repair",
         designDiscipline: "designDiscipline2",
         status: 2,
         location: {
-            lat: 52.558125,
-            lng: -1.821516,
+            lat: 48.6345,
+            lng: -1.927523,
           },
         likelihood:3,
         severity:4,
         risk:7,
-        projectId:"5f4eec89cc853f32ec03243e"         
+        projectId:"5f59f053a94acd5aa0e4ebec"         
     },
     {
-        title: "title3",
+        title: "Big Ben",
         riskId: "riskId3",
-        description:"discription3",
+        description: "Renew",
         designDiscipline: "designDiscipline3",
         status: 2,
         location: {
-            lat: 52.652311,
-            lng: -1.126321,
+            lat: 47.4266,
+            lng: -1.463867,
           },
         likelihood:3,
         severity:4,
         risk:7,
-        projectId:"5f4eec89cc853f32ec03243e"
+        projectId:"5f59f053a94acd5aa0e4ebed"
     },
   
```

Comment:
Can you add some more risks which have a range of likelihood and severity values, so that we can demonstrate the colour coding of risks on the map